.bazelrc

```diff
@@ -40,8 +40,7 @@ common:remote-shared --remote_timeout=600
 common:remote-shared --remote_download_minimal
 common:remote-shared --jobs=100
 common:remote-shared --verbose_failures
-common:remote-shared --platforms=//platforms:linux_x86_64
-common:remote-shared --extra_execution_platforms=//platforms:linux_x86_64
+common:remote-shared --extra_execution_platforms=@buildbuddy_toolchain//:platform_linux
```

Comment:
Without a target platform, we will not be able to build remotely from a MacOS host.
If you are planning to add a `remote-linux-arm64` config in `.bazelrc`, I think we can move these flags from `remote-shared` to `remote` so that `remote-linux-arm64` can inherit from `remote-shared`

Reply:
Done!